processors/base/helpers/__init__.py

```diff
@@ -86,6 +86,10 @@ def clean_identifiers(identifiers):
     }
     result = {}
     for key, value in identifiers.items():
+        try:
+            value = re.sub(r'\s', '', value)
+        except TypeError:
+            pass
```

Comment:
Is there a way to know for which identifiers is the space safe to remove or I should just remove this code and leave the spaces alone?